### DISCUSSION TITLE
How can I change the h,j,k,l to h,a,e,i?

### DISCUSSION THREAD

**Discussion by azizsayfiddin** [orhun/gpg-tui#261] [Q&A] [answered]
I use the alternative keyboard layout graphite-angle-kp, which looks like this:
![image](https://github.com/user-attachments/assets/ff2975ee-800c-459a-8cab-61fc26263d69)
And I want to change the h,j,k,l to h,a,e; how can I do that?
I tried to run it with this config: ```[general]
key_bindings = [
  {keys = [ "a" ], command = "down"},
  {keys = [ "e" ], command = "up"},
  {keys = [ "i" ], command = "right"},
]```, but it says that there aren't such commands.

**Comment by orhun** [comment, marked as answer]
Hey!

You can check out the list of commands and their examples from [`COMMANDS.md`](https://github.com/orhun/gpg-tui/blob/master/COMMANDS.md)

So your config should look a little something like this:

```toml
[general]
key_bindings = [
  { keys = [
    "a",
  ], command = ":scroll down 1" },
  { keys = [
    "e",
  ], command = ":scroll up 1" },
]
```

**Reply by azizsayfiddin** [reply]
I usually read only --help, manpages and tldr pages. From now on, I will consider looking at the documents in the github repo.
